lib/httplib/httpheaders.go

```diff
@@ -21,10 +21,66 @@ package httplib
 import (
 	"fmt"
 	"net/http"
+	"sort"
 	"strings"
 	"time"
+
+	"golang.org/x/exp/maps"
 )
 
+var defaultContentSecurityPolicy map[string]string = map[string]string{
+	"default-src": "'self'",
+	// specify CSP directives not covered by `default-src`
+	"base-uri":        "'self'",
+	"form-action":     "'self'",
+	"frame-ancestors": "'none'",
+	// additional default restrictions
+	"object-src": "'none'",
+	// auto-pay plans in Cloud use stripe.com to manage billing information
+	"script-src": "'self' https://js.stripe.com",
```

Comment:
How can we get Stripe removed from the default? It's only needed for Teleport Team plans on Cloud. Shouldn't be part of OSS or Ent...

Reply:
@reedloden The cloud team has a [goal this quarter](https://gravitational.slab.com/posts/teleport-cloud-goals-fy-24-q-2-ytrpyrfo#hhy9e-stripe-csp-fix) to minimize its use. Looks like Isaiah is just moving things around.